conf/aws/quartz.properties

```diff
@@ -0,0 +1,3 @@
+org.quartz.scheduler.instanceName = LocalJobScheduler
```

Comment:
seems to be the same as the original `quartz.properties` file, can we just use that?